pandas/core/ops/array_ops.py

```diff
@@ -474,9 +473,4 @@ def _maybe_upcast_for_op(obj, shape: Shape):
         #  np.timedelta64(3, 'D') / 2 == np.timedelta64(1, 'D')
         return Timedelta(obj)
 
-    elif isinstance(obj, np.ndarray) and obj.dtype.kind == "m":
-        # GH#22390 Unfortunately we need to special-case right-hand
```

Comment:
can this comment be retained somewhere useful?

Reply:
Any suggestion where? 
I could add it in the docstring of `ensure_wrapped_if_datetimelike` (although that's not only used for ops). I could also leave it right here where it was, but amend it with saying that this already has been done by calling `ensure_wrapped_if_datetimelike` before.

Reply:
i guess adjacent to the ensure_wrapped_if_datetimelike call before which this is called

Reply:
Moved it slightly reworded to `arithmetic_op`. Does that look good?

Reply:
great, thanks